Ports/cowsay/package.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='cowsay'
+version='3.04'
+depends=('perl5')
+useconfigure='false'
+files="https://github.com/tnalpgge/rank-amateur-cowsay/archive/refs/tags/cowsay-${version}.tar.gz cowsay-${version}.tar.gz d8b871332cfc1f0b6c16832ecca413ca0ac14d58626491a6733829e3d655878b"
+workdir="rank-amateur-cowsay-cowsay-${version}/"
+
+prefix='/usr/local'
+bin_path="${prefix}/bin"
+cow_path="${prefix}/share/cows"
+perl_executable="${bin_path}/perl"
+
+build() {
+    run sed -i "s#%PREFIX%#${prefix}#" cowsay
+    run sed -i "s#%BANGPERL%#!${perl_executable}#" cowsay
+}
+
+install() {
+    run mkdir -p "${SERENITY_INSTALL_ROOT}/${bin_path}/"
+    run cp cowsay "${SERENITY_INSTALL_ROOT}/${bin_path}/"
+    run mkdir -p "${SERENITY_INSTALL_ROOT}/${cow_path}/"
+    run cp -r cows/ "${SERENITY_INSTALL_ROOT}/${cow_path}/"
```

Comment:
Perhaps use `install` instead of `mkdir` and `cp` in both places?

Reply:
I decided against doing this because I'm not sure how portable `install` is between Linux and Mac / BSD systems (I believe there are differences in how the `-D`/`-d` options work.) I could only find [one example](https://github.com/SerenityOS/serenity/blob/b5a728ae5f62c4fabf69dc7fb39883fcb210a69e/Ports/alpine/package.sh#L35-L43) of a port using `install` and its use is made slightly annoying that we have an `install()` function in the script already.